help.md

```diff
@@ -22,15 +22,13 @@ Additionally the plugin can be installed by manually building the hpi file and u
 
 ## Technical Details
 
-### Development
+Additional documentation for installing and configuring the Jenkins plugin can be found on the [Rapid7 help pages](https://insightappsec.help.rapid7.com/docs/jenkins-integration).
 
-To run the plugin locally, cd to the root directory and invoke:
+### Installation
```

Comment:
There is an installation section at the top of the readme already?

Reply:
Under `Contributions` of the help.md (used by the extension listing library) we mention referencing the README/repo.  Trying to keep developer specific content from the actual listing as it is intended for end users.